Source/DiabloUI/art.cpp

```diff
@@ -30,23 +29,23 @@ bool LoadPcxMeta(SDL_RWops *handle, int &width, int &height, std::uint8_t &bpp)
 }
 
 bool LoadPcxPixelsAndPalette(SDL_RWops *handle, int width, int height, std::uint8_t bpp,
-    uint8_t *buffer, std::size_t bufferPitch, SDL_Color *palette)
+    uint8_t *buffer, std::ptrdiff_t bufferPitch, SDL_Color *palette)

```

Comment:
`bufferPitch` cannot be negative, it's not a difference, `ptrdiff_t` doesn't seem appropriate here.

Reply:
It's called with `SDL_Surface::pitch` which is currently int, I agree an unsigned type would be more appropriate (and width/height would also be better as unsigned types) but given the use is tied to SDL types I figured this was the best fit for the current caller.